### DISCUSSION TITLE
Multiple instances for family members?

### DISCUSSION THREAD

**Discussion by GrowingUpFisher** [ananthakumaran/paisa#250] [Q&A]
Hi, 

I would like to separately track the finances for my family members. 

Is there any way to setup separate portfolios/instances for family members without spinning up a new server instance ?

Thanks,
Fisher

**Comment by Ananthu26** [comment]
Untill Next Profile/User update try Maybe ,it supports multiple user accounts to have their own profiles. and admin, you can control new user registrations by configuring the REQUIRE_INVITE_CODE environment variable or through the web interface.

https://github.com/maybe-finance/maybe
